qiskit/providers/ibmq/job/ibmqjob.py

```diff
@@ -190,11 +190,17 @@ def properties(self):
         """Return the backend properties for this job.
 
         Returns:
-            BackendProperties: the backend properties used for this job.
+            BackendProperties: the backend properties used for this job, or None if
+                properties are not available.
         """
         self._wait_for_submission()
 
         properties = self._api.job_properties(job_id=self.job_id())
+
+        # Backend properties of a job might not be available if the job hasn't
+        # completed. This is to ensure the properties returned are up to date.
```

Comment:
Can you find a way to include this or a similar clarification in the actual docstring? In this particular issue, it feels important for the users to be aware that a job's properties will not be available in all cases (as summarized in the issue), and the docstrings are arguably the best way to notify them (as opposed to inline comments in the code, with are a great practice - but more oriented towards developers and future maintainers).